### PR TITLE
Add initial tests for the help behavior

### PR DESCRIPTION
**Pull request by ryzokuken**

### CODE REVIEW

**Comment by ryzokuken** [comment]
@jywarren @ananyo2012 Do the tests seem more appropriate now?

File: utils.js [hunk 1]
```diff
@@ -42,5 +42,7 @@ function messageResponse (name, message) {
 
 module.exports = {
   parseMessage,
-  messageResponse
+  messageResponse,
+  printGeneralHelp,
```
Comment (ananyo2012) [inline utils.js]: Do we need to make this functions public? Can't we do this with `messageResponse` function?

Comment (ryzokuken) [inline utils.js]: I personally feel using messageResponse would be too high level. If you say, I would change the tests to use messageResponse. I will submit a commit which these changes later today.

**Comment by ryzokuken** [comment]
@ananyo2012 I think for this one, these seem okay. Please help me figure out a way to test the greet behavior too. (Or should we not?)

**Comment by ananyo2012** [comment]
For that we need to mock a connection for irc. https://github.com/martynsmith/node-irc/blob/master/test/helpers.js uses a `MockIrcd` class to do so. Well we can figure out that. For now I am merging this one. Thanks for the great work!

**Pull request merged**